project/.travis/before_script_test.sh.twig

```diff
@@ -1,3 +1,11 @@
 #!/usr/bin/env bash
 set -ev
 
+{% if repository_name is same as('SonataDoctrinePhpcrAdminBundle') %}
+CONSOLE=vendor/symfony-cmf/testing/bin/console
+
+export KERNEL_CLASS="Sonata\\DoctrinePHPCRAdminBundle\\Tests\\Fixtures\\App\\Kernel"
+echo '+++ create PHPCR +++'
+${CONSOLE} doctrine:phpcr:init:dbal --drop --force -vvv
+${CONSOLE} doctrine:phpcr:repository:init -vvv
+{% endif %}
```

Comment:
Wouldn't it be nicer to just have something along the lines of
```bash
if [ -f before_script_test.local.sh ]
then
    ./before_script_test.local.sh
fi
```
(that's probably not the correct syntax / doesn't work from any working directory but you get the idea)

Reply:
i think that would be better to separate responsibilities between the repos, yes.

do we have such "hooks" anywhere else? we should be consistent with them, ideally.

Reply:
I don't think we do.

Reply:
We must take care to not be tempted to write quickly some script that could be common.

But yeah, I would also prefer a *.local.sh method for that case.

Reply:
BTW, the @greg0ire method looks good to me.

Maybe place the condition on both places:

- Before the main script execution
- At the end of the main script execution

WDYT?

Reply:
yeah, if we have before_script_test.local.sh we should likely also have after_script_test.local.sh :-D

Reply:
I created a PR https://github.com/sonata-project/dev-kit/pull/465, shall I change sth. ?